blueprintapi/oscal/component.py

```diff
@@ -161,6 +162,17 @@ def control_ids(self) -> List[str]:
             }
         )
 
+    @property
+    def catalog_versions(self) -> List[Catalog.Version]:
+        versions = set()
+        for item in self.control_implementations:
+            try:
+                versions.add(Catalog.Version(item.description))
```

Comment:
Yay, ok cool- this is where we're creating the "real" list of catalogs that a component belongs to